src/paths/all_shortest_paths.c

```diff
@@ -334,3 +340,18 @@ igraph_error_t igraph_get_all_shortest_paths(const igraph_t *graph,
 
     return IGRAPH_SUCCESS;
 }
+
+/**
+ * \function igraph_get_all_shortest_paths
+ * \brief All shortest paths (geodesics) from a vertex.
+ *
+ * \ref igraph_get_all_shortest_paths() without a cutoff.
```

Comment:
Did you mean the following?

```suggestion
 * \ref igraph_get_all_shortest_paths_cutoff() without a cutoff.
```

Can we keep the full original docs here please, especially since the new function will be "experimental"?